tidy3d/web/webapi.py

```diff
@@ -202,15 +158,10 @@ def get_run_info(task_id: TaskId):
         Average field intensity normlized to max value (1.0).
         Is ``None`` if run info not available.
     """
-    try:
-        token = get_s3_sts_token(task_id, "output/solver_progress.csv")
-        client = token.get_client()
-        progress = client.get_object(Bucket=token.get_bucket(), Key=token.get_s3_key())["Body"]
-        progress_string = progress.read().split(b"\n")
-        perc_done, field_decay = progress_string[-2].split(b",")[:2]
-        return float(perc_done), float(field_decay)
-    except Exception:  # pylint:disable=broad-except
-        return None, None
+    task = SimulationTask.get(task_id)
```

Comment:
we introduce a extra call to get running info.
considering this call will be called in python side every seconds.
it is not worth to add such extra call.

Reply:
So I guess the idea here is that after the simulation task is uploaded, we don't need to keep checking if it exists from the monitor and other steps?

Reply:
in order to get running status, the code have 2 webapi: